### DISCUSSION TITLE
Noise reduction icons

### DISCUSSION THREAD

**Discussion by ShaunHalliday** [twilio-labs/paste#2587] [Request: Icon] [answered]
Hey all,

We are trialling the new technology Twilio added around noise reduction, and we require icons for this when active and not active. Please see the below for requirement.

![noise-reduction-icon-on](https://user-images.githubusercontent.com/85227673/182795461-8da8d46e-c594-4501-b3fe-9475c0e3087a.svg)

![noise-reduction-icon-off](https://user-images.githubusercontent.com/85227673/182795464-08faeb61-7d9f-4ab3-91aa-01c735f00b28.svg)

Kind Regards,
Shaun

**Comment by loreina** [comment]
Hi @ShaunHalliday,

Thanks for creating a discussion! Do you mind sharing more context around your use case for these icons? Screenshots or a Figma file would be really helpful. Our team is looking to better understand the use case and need to consider if these are icons to be added to Paste. Thank you!

**Reply by ShaunHalliday** [reply]
Hi Loreina,

The system these icons will be used on is a webRTC conferencing platform. 
From a use case point of view-  These icons will appear next to a mute button, when a speaker is active on the system. The icons will be displayed, showing whether noise cancellation is active or inactive. I have put together figma designs at the following link. If you run the prototype I have put in place a tooltip on hover stating whether the noise cancellation is active or not, feel free to provide feedback whether this or a text alongside the icon would be better:

https://www.figma.com/file/mTiBA9yzkmPB4PuhcM2iJz/Twilio-Noise-Cancellation?node-id=411%3A2

**Comment by loreina** [comment, marked as answer]
Hi @ShaunHalliday, thanks for sharing the designs.

We have a noise reduction in the works on the product side, though unfortunately don't have a timeline to share right now. In the meantime, you can create your own custom icon by using our [SVG to React package](https://github.com/twilio-labs/svg-to-react) or copying the source of an Icon and changing the SVG in it.

Someone on the team will follow up when a noise reduction icon is available. Thank you!

**Comment by ShaunHalliday** [comment]
Hi Loreina,

Thanks for the advice, will take this to our development team and look out for icon in future.

Thank you again,
Shaun
